setup.py

```diff
@@ -59,6 +60,9 @@ def package_files(package_dir, subdirectory):
         'aipy>=3.0',
         'click'
     ],
+    "extras_require" : [
+        'bda @ git+git://github.com/HERA-Team/baseline_dependent_averaging'
```

Comment:
this should be a dictionary like {"bda": "bda @ git+git://github.com/HERA-Team/baseline_dependent_averaging"}